src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+import _ from 'lodash';
+
+@Component({
+  selector: 'cd-feedback',
+  templateUrl: './feedback.component.html',
+  styleUrls: ['./feedback.component.scss']
+})
+export class FeedbackComponent {
+  title = 'Feedback';
+  project: any = ['dashboard', 'block', 'objects', 'file_system', 'ceph_manager', 'orchestrator', 'ceph_volume', 'core_ceph'];
+  tracker: any = ['bug', 'feature'];
+  // severity: any = ['minor', 'major', 'critical']
+  feedbackForm = new FormGroup({
+    project: new FormControl('', Validators.required),
+    tracker: new FormControl('', Validators.required),
+    subject: new FormControl('', Validators.required),
+    description: new FormControl('', Validators.required),
+  });
+
+  get f() {
+    return this.feedbackForm.controls;
+  }
+
+  onSubmit() {
+    fetch('/ui-api/feedback', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/vnd.ceph.api.v1.0+json',
+        'Accept-Language': 'en-US,en;q=0.5',
+        'Connection': 'keep-alive',
+      },
```

Comment:
We have api service calls separated on this folder https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/shared/api. 

Do you think its possible to clean this up?

Reply:
done